signature/simple/signer.go

```diff
@@ -0,0 +1,104 @@
+package simple
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/containers/image/v5/docker/reference"
+	internalSig "github.com/containers/image/v5/internal/signature"
+	internalSigner "github.com/containers/image/v5/internal/signer"
+	"github.com/containers/image/v5/signature"
+	"github.com/containers/image/v5/signature/signer"
+)
+
+// simpleSigner is a signer.Signer implementation for simple signing signatures.
```

Comment:
Nit: Could we embed GPG in the name?  It would be more expressive to me when reading.

Reply:
Should it even be `signature/simple`? `signature/simplesigning` would be consistent with `internal/signature.SimpleSigning`.

Reply:
Moved to `signature/simplesigning` now, but that’s not intended to stop the search for another name.

Reply:
`simplesigning` SGTM
It's somehow a known terminology in our universe.